zulipterminal/ui_tools/messages.py

```diff
@@ -152,9 +152,12 @@ def stream_header(self) -> Any:
         assert self.stream_id is not None
         color = self.model.stream_dict[self.stream_id]["color"]
         bar_color = f"s{color}"
+        stream_access_type = self.model.stream_access_type(self.stream_id)
+        stream_icon = STREAM_ACCESS_TYPE[stream_access_type]["icon"]
         stream_title_markup = (
             "bar",
             [
+                (bar_color, f" {stream_icon} "),
                 (bar_color, f"{self.stream_name} {STREAM_TOPIC_SEPARATOR} "),
```

Comment:
It would be nice to combine these two lines, but it does put it over the line limit, so this is fine as it is :+1: